manager/src/main/java/io/gingersnapproject/metrics/micrometer/CacheManagerMicrometerMetrics.java

```diff
@@ -0,0 +1,80 @@
+package io.gingersnapproject.metrics.micrometer;
+
+import java.time.Duration;
+import java.util.EnumMap;
+import java.util.concurrent.TimeUnit;
+
+import io.gingersnapproject.metrics.CacheAccessRecord;
+import io.gingersnapproject.metrics.CacheManagerMetrics;
+import io.micrometer.core.instrument.MeterRegistry;
+import io.micrometer.core.instrument.Timer;
+
+/**
+ * A {@link CacheManagerMetrics} implementation that uses {@link MeterRegistry} to store metrics information.
+ */
+public class CacheManagerMicrometerMetrics implements CacheManagerMetrics {
+
+    private final EnumMap<TimerMetric, Timer> timers = new EnumMap<>(TimerMetric.class);
+
+    private static Timer createTimer(MeterRegistry registry, String mame, String description) {
+        return Timer.builder(mame)
+                .description(description)
+                .tags("gingersnap", "cache_manager")
+                .publishPercentileHistogram()
+                // TODO keep or allow to be configured?
+                // pre-defined histogram bucket so we can see the number of request with latency <= the values below
+                .serviceLevelObjectives(
+                        Duration.ofMillis(1),
+                        Duration.ofMillis(5),
+                        Duration.ofMillis(10),
+                        Duration.ofMillis(50),
+                        Duration.ofMillis(100),
+                        Duration.ofMillis(500))
+                // TODO keep or configure?
+                // By default, micromenter creates a tons of bucket with values > 1second. We don't need them.
+                .maximumExpectedValue(Duration.ofSeconds(1))
```

Comment:
I would say that it's overkill making these values configurable is overkill for now given that we're the only consumer of the Metrics.